src/components/v2/Table/index.tsx

```diff
@@ -41,92 +23,88 @@ export const Table = ({
   rowKeyIndex,
   className,
   gridTemplateColumns,
+  gridTemplateRowsMobile = '1fr',
+  isMobileView,
```

Comment:
Do we need this prop if we handle responsive styles with css?

Reply:
Nope, deleted.